proto/v1/setting_service.proto

```diff
@@ -243,6 +243,9 @@ message WorkspaceProfileSetting {
 
   // The workspace database change mode.
   DatabaseChangeMode database_change_mode = 11;
+
+  // The workspace mode, either Console or SQL Editor.
+  WorkspaceMode workspace_mode = 12;
```

Comment:
The same as   DatabaseChangeMode database_change_mode = 11;?

Reply:
consider DatabaseChangeMode is still a separate setting to enable SQL Editor standard mode when WorkspaceMode is "CONSOLE" so I didn't use it

Reply:
What's the difference?

Reply:
DatabaseChangeMode can be "EDITOR" (to enable standard mode) when WorkspaceMode is "CONSOLE"